gui/packages/desktop/src/main/index.js

```diff
@@ -65,8 +66,25 @@ const ApplicationMain = {
   _oldLogFilePath: (null: ?string),
   _quitStage: ('unready': AppQuitStage),
 
-  _tunnelState: defaultTunnelStateTransition(),
-  _settings: defaultSettings(),
+  _tunnelState: { state: 'disconnected' },
```

Comment:
FYI since we do not dupe the same defaults in renderer anymore due to synchronous call from renderer  to main, I've removed the functions returning blank states for both settings and tunnelState.